### ISSUE TITLE
When saving bookmarks in Firefox on version 0.7, tag is not added

### ISSUE THREAD

**Issue by sunjam** [damko/freedommarks-browser-webextension#5]
When saving a bookmark via the Freedommarks 0.7 firefox extension, url and description sync to my 12.0.5 server, but tags are not saved.

**Comment by damko** [comment]
This is very weird. Can you save tags from the web interface (in nextcloud) ?

**Comment by sunjam** [comment]
Just having a problem with Freedommarks.  Any specific logs I should share?

**Comment by damko** [comment]
About logs: I'll give you instructions in the next days. I need to check something on the nextcloud side.
If you upgraded from a previous Freedommarks version, please try this:
* remove Freedommarks
* restart firefox
* reinstall Freedommarks

As far as I can tell, you are the only one having this kind of issue and, to be honest, I believe that there is something wrong on the nextcloud installation on server side. Maybe nextcloud bookmarks has been updated and I need to adjust something in Freedommarks. However I've planned to publish a new release in a couple of weeks. Hold tight meanwhile.

Thanks for reporting

**Comment by bepolymathe** [comment]
Hello, 

I have the same problem. Firefox 59

**Comment by damko** [comment]
bug confirmed and partially diagnosed. It definitely has something to do with the recent changes in the API server side. It was working on nextcloud 12.x but after upgrading to 13.1 tags don't get saved anymore. I'll look into it asap

**Comment by n1mh** [comment]
Hi,

I have the same problem and it's probably related with nextcloud 13. Before that version it worked great with tags.

**Comment by bepolymathe** [comment]
It is true that this is a disabling problem. It would be great if you could do something...
Thank you in advance.

**Comment by damko** [comment]
yes definitely related to changes in the API for bookmark-apps version 0.11.

I'm looking into it now

**Comment by damko** [comment]
I'm going to upload v.0.0.8 which fixes this bug. Tags now can be added to a new bookmark via FreedomMarks.
Be aware though that updating already existent tags won't work. I'm not yet sure what is this due too. I'll diagnose in the next days.

**Comment by bepolymathe** [comment]
Hello, 

I confirm it works.
